adal/src/main/java/com/microsoft/aad/adal/AcquireTokenRequest.java

```diff
@@ -701,6 +707,10 @@ private void waitingRequestOnError(final CallbackHandler handler, final Authenti
         }
     }
 
+    public final boolean isOutageMode() {
+        return mAuthContext.getExtendedLifetimeEnabled();
```

Comment:
don't spread the isOutageMode around, also if you are just going to return id extendedLifetime is enabled, call that function